.github/workflows/cd-dgraph.yml

```diff
@@ -39,15 +39,15 @@ jobs:
       - name: Make Linux Build
         run: make dgraph DGRAPH_VERSION=${{ env.DGRAPH_RELEASE_VERSION }}
       - name: Generate SHA for Linux Build
-        run: sha256sum dgraph/dgraph | cut -c-64 > dgraph/dgraph-linux-amd64.sha256
+        run: cd dgraph && sha256sum dgraph | cut -c-64 > dgraph-linux-amd64.sha256 && mv dgraph-linux-amd64.sha256 ..
       - name: Tar Archive for Linux Build
-        run: tar -zcvf dgraph/dgraph-linux-amd64.tar.gz dgraph/dgraph
+        run: cd dgraph && tar -zcvf dgraph-linux-amd64.tar.gz dgraph && mv dgraph-linux-amd64.tar.gz ..
```

Comment:
I think you can remove the `&& mv dgraph-linux-amd64.tar.gz ..`

reason: you can upload the created artifact from the original path `dgraph/dgraph-linux-amd64.tar.gz`